src/v2/cookbook/form-validation.md

```diff
@@ -333,21 +333,34 @@ We set up the total value as a computed value, and outside of that bug I ran int
 
 ## Server-side Validation
 
-In my final example, we built something that makes use of Ajax to validate at the server. The form will ask you to name a new product and will then check to ensure that the name is unique. We wrote a quick [OpenWhisk](http://openwhisk.apache.org/) serverless action to do the validation. While it isn't terribly important, here is the logic:
+In my final example, we built something that makes use of Ajax to validate at the server. The form will ask you to name a new product and will then check to ensure that the name is unique. We wrote a quick [Netlify](https://netlify.com/) serverless action to do the validation. While it isn't terribly important, here is the logic:
 
 ``` js
-function main(args) {
-    return new Promise((resolve, reject) => {
-        // bad product names: vista, empire, mbp
-        const badNames = ['vista', 'empire', 'mbp'];
-
-        if (badNames.includes(args.name)) {
-          reject({error: 'Existing product'});
-        }
+exports.handler = async (event, context) => {
+  
+    const badNames = ['vista', 'empire', 'mbp'];
+    let name = event.queryStringParameters.name;
+
+    if (badNames.includes(name)) {
+      return { 
+        statusCode: 200,         
+        headers:{
+          'Access-Control-Allow-Origin':'*'
```

Comment:
Since this function is supposed to have the same origin as the code in our context, this shouldn't be necessary. Same goes with the "success" logic.

Reply:
But then it can't be run in the CodePen.

Reply:
The CodePen version can keep the header but for a normal context (hence "in our context" in my original comment) the header is redundant.